python/lsst/obs/base/cameraMapper.py

```diff
@@ -423,6 +423,15 @@ def setMethods(suffix, mapImpl=None, bypassImpl=None, queryImpl=None):
                                        bypassImpl=lambda datasetType, pythonType, location, dataId:
                                        afwImage.Filter(
                                            afwImage.readMetadata(location.getLocationsWithRoot()[0])))
+                            setMethods("bbox", bypassImpl=lambda dsType, pyType, location, dataId:
```

Comment:
All of the examples above use `datasetType` and `pythonType`: that's better than an unnecessary abbreviation. And the same below.